src/codegen/inserter.cpp

```diff
@@ -34,6 +34,8 @@ void Inserter::Init(storage::DataTable *table,
 char *Inserter::AllocateTupleStorage() {
   location_ = table_->GetEmptyTupleSlot(nullptr);
 
+  PELOTON_ASSERT(location_.IsNull() == false);
```

Comment:
Same question as above. Why was this added? Ideally, I believe these checks should be handled at a higher layer, probably the `DataTable`.